src/api/notification/content-types/notification/schema.json

```diff
@@ -50,6 +50,28 @@
         "shared.contact",
         "shared.content"
       ]
+    },
+    "date": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": false
+        }
+      },
+      "type": "datetime",
+      "required": true
+    },
+    "type": {
```

Comment:
Discussed with Fadwa (designer), we should be able to use any icon from design system
So please use "iName" prop instead of "Type" and put there all the icons
You can copy this field from Button component

<img width="1242" alt="image" src="https://github.com/BliqBV/strapi-cloud-bliq-app/assets/151524383/27c26462-80d6-4f84-9716-0101e28fae4b">

Reply:
Type field is replaced with iName field.